### PR TITLE
YLP-2071 change the question mark icon to accept button

### PR DESCRIPTION
**Pull request by Xwdrblue**
## Checklist
- [x] PR name contains the Jira ticket number
- [x] Changelog updated
- [x] Code tested (unit tests and integration tests)
- [x] No console errors/warnings in integration tests
- [x] No translation files updated (except for English files)
- [x] Code formatted on every file
- [x] Copyright added on every file
- [x] End-to-end tests successful on CI

### CODE REVIEW

File: packages/yourloops/pages/notifications/notification.tsx [hunk 1]
```diff
@@ -257,7 +251,7 @@ export const Notification = (props: NotificationProps): JSX.Element => {
   }
 
   const onDecline = async (): Promise<void> => {
-    setInProgress(true)
+    // setInProgress(true)
```
Comment (celia-rouis) [inline packages/yourloops/pages/notifications/notification.tsx:254]: **🔨 issue:** ‏Commented code should be removed. This gives the impression that the PR is not ready for review yet, is it the case?

File: packages/yourloops/pages/notifications/notification.tsx [hunk 2]
```diff
@@ -294,19 +288,22 @@ export const Notification = (props: NotificationProps): JSX.Element => {
...
         {isAMonitoringInvitation && displayMonitoringTerms && notification.target &&
-          <MonitoringConsentDialog onAccept={acceptTerms} onCancel={() => { setDisplayMonitoringTerms(false) }}
+          <MonitoringConsentDialog onAccept={acceptTerms} onCancel={() => {
+            setDisplayMonitoringTerms(false)
+          }}
                                    teamName={notification.target.name} />
         }
         {props.userRole === UserRoles.caregiver && notification.type === NotificationType.careTeamProInvitation ? (
-          <IconButton
+          <Button
```
Comment (celia-rouis) [inline packages/yourloops/pages/notifications/notification.tsx:297]: **🎯 suggestion (blocking):** ‏Here we have almost the same button defined twice in the ternary condition. The ternary condition could be removed, and a method `handleAcceptButtonClick()` should be created to handle the difference of behaviour between the case when a caregiver is invited to become a HCP, and the other cases

File: packages/yourloops/components/switch-role/consent-dialog.tsx [hunk 3]
```diff
@@ -98,7 +98,7 @@ function SwitchRoleConsentDialog(props: SwitchRoleConsentDialogProps): JSX.Eleme
           {t('button-decline')}
         </Button>
         <Button
-          id="switch-role-consent-dialog-button-accept"
+          data-testid='switch-role-consent-dialog-button-accept'
```
Comment (celia-rouis) [inline packages/yourloops/components/switch-role/consent-dialog.tsx:101]: **🎯 suggestion (non-blocking):** ‏Double quotes should be used for strings in HTML code

File: packages/yourloops/pages/notifications/notification.tsx [hunk 4]
```diff
@@ -163,11 +161,7 @@ export const NotificationSpan = ({ notification, id }: NotificationSpanProps): J
   return <span id={id} className={`${classes.notificationSpan} notification-text`}>{notificationText}</span>
 }
 
-const NotificationIcon = ({
-  id,
-  type,
-  className
-}: { id: string, type: NotificationType, className: string }): JSX.Element => {
+const NotificationIcon = ({ id, type, className }: { id: string, type: NotificationType, className: string }): JSX.Element => {
```
Comment (fatb38) [inline packages/yourloops/pages/notifications/notification.tsx:164]: **🤓 nitpick:** ‏The type could be an interface instead for a better readability

File: packages/yourloops/pages/notifications/notification.tsx [hunk 5]
```diff
@@ -294,32 +291,26 @@ export const Notification = (props: NotificationProps): JSX.Element => {
...
-            disableElevation
-            className={`${classes.buttonAccept} notification-button-accept`}
-            disabled={inProgress}
-            onClick={onOpenInvitationDialog}
-          >
-            {t('button-accept')}
-          </Button>
-        )}
+        <Button
+          id={`notification-button-accept-${id}`}
```
Comment (fatb38) [inline packages/yourloops/pages/notifications/notification.tsx:303]: **🎯 suggestion (non-blocking):** ‏Could be removed unused id

Comment (Xwdrblue) [reply packages/yourloops/pages/notifications/notification.tsx:303]: i need this for unitary test

Comment (Xwdrblue) [reply packages/yourloops/pages/notifications/notification.tsx]: but i replace id by `id='notification-button-accept`

**Review by fatb38 — APPROVED** [review]

**Pull request merged**